.github/workflows/curl-spike.yml

```diff
@@ -0,0 +1,13 @@
+name: troubleshooting curl issue
+
+on:  workflow_dispatch
+
+
+jobs:
+  which-curl:
+    name: Which curl
+    runs-on: ubuntu-latest
+    steps:
+      - name: check to see where curl lives
+        run: |
+          which curl
```

Comment:
Should we run this delegated to localhost as well? Maybe both with and without that attribute?

Reply:
Scratch that I was thinking of ansible YML, not an action